tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -706,21 +706,21 @@ def test_get_random_state_no_cupy_no_chainer_seed(self):
 
     def test_get_random_state_no_cupy_with_chainer_seed(self):
         os.environ.pop('CUPY_SEED', None)
-        os.environ['CHAINER_SEED'] = '5'
+        os.environ['CHAINER_SEED'] = str(5)
```

Comment:
This use of `str` is not necessary. Could you revert it? (same for other changes)